scripts/translation/xliffmerger.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+import xml.etree.ElementTree as ET
+import os
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+
+def delete_trans_units(body_element, trans_units, units_to_remove):
+    to_delete = [tr_unit for tr_unit in trans_units if tr_unit.attrib['id'] in units_to_remove]
+    for item in to_delete:
+        body_element.remove(item)
+
+
+def add_trans_units(body_element, trans_units, units_to_add):
+    to_add = [tr_unit for tr_unit in trans_units if tr_unit.attrib['id'] in units_to_add]
+    for item in to_add:
+        item.set('state', 'new')
+        body_element.append(item)
+
+
+def update_trans_units(body_element, trans_units, id_new_value_dict):
+    to_update = [tr_unit for tr_unit in trans_units if tr_unit.attrib['id'] in id_new_value_dict]
+    for trans_unit in to_update:
+        trans_unit.set('state', 'needs-adaptation')
```

Comment:
`state` is an attribute for "&lt;target&gt;". Here it seems you set it on &lt;trans-unit&gt;

http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#state